test/cli/fixtures/custom-runner/custom-runner.test.ts

```diff
@@ -0,0 +1,53 @@
+import {describe, expect, test, type TestAPI} from 'vitest'
+
+const myTest = test as TestAPI<{result: (name: string) => Promise<unknown>}>
+
+describe('await multiple prior test result', () => {
+	myTest('first', async () => {
+		expect(1).toBe(1)
+		return 1
+	})
+
+	myTest('second', async () => {
+		expect(1).toBe(1)
+		return 2
+	})
+
+	myTest('third', async () => {
+		expect(1).toBe(1)
+		return 3
+	})
+
+	myTest.only('with await', async ({result}) => {
```

Comment:
Isn't CI failure due to `only? Locally it should be fine, but Vitest disables `allowOnly: false` by default on CI.

> [Vitest] Unexpected .only modifier. Remove it or pass --allowOnly argument to bypass this error

Reply:
thanks, fixed by configuring it to `true`